docs/docs/get_started/how-to-deploy/overview.md

```diff
@@ -1,9 +1,9 @@
 ---
 title: How to Deploy
 id: how_to_deploy
-layout: docs
 description: Secretless Broker Documentation
 permalink: docs/get_started/how-to-deploy/overview.html
+redirect_to: https://docs.secretless.io/Latest/en/Content/Resources/_TopNav/cc_Home.htm
```

Comment:
Let's change this to https://docs.secretless.io/Latest/en/Content/Get%20Started/using-conjur.htm